src/AuthorizedSocketConnection.ts

```diff
@@ -79,6 +81,7 @@ export default class AuthorizedSocketConnection<TContext, TCredentials> {
   ) {
     this.socket = socket;
     this.config = config;
+    this.clientId = socket.client.id;
```

Comment:
@taion I think this is accurate?

Reply:
you can simplify slightly i think?

```suggestion
    this.clientId = socket.id;
```